sdk/servicebus/azure-servicebus/azure/servicebus/_servicebus_sender.py

```diff
@@ -78,9 +78,9 @@ def _create_attribute(self, **kwargs):
         self._entity_uri = f"amqps://{self.fully_qualified_namespace}/{self._entity_name}"
         # TODO: What's the retry overlap between servicebus and pyamqp?
         self._error_policy = self._amqp_transport.create_retry_policy(self._config)
-        self._name = kwargs.get("client_identifier",f"SBSender-{uuid.uuid4()}")
+        self._name = kwargs.get("client_identifier") or f"SBSender-{uuid.uuid4()}"
```

Comment:
couldn't comment on line 75, but we can remove `object`. We can just type out the kwarg here as well.